etl/message/forms.py

```diff
@@ -57,7 +56,10 @@ def __init__(self, *args, **kwargs):
     def _prepare_message(self) -> Dict[str, str]:
         """Prepare message for Form-based validation."""
         msg_data = {}
-        message = p.parsestr(self.archive.format_message(self.archive_msg))
+        rfc822 = self.archive.format_message(self.archive_msg)
+        # remove bad header line before creating EmailMessage
+        rfc822 = rfc822.replace("Microsoft Mail Internet Headers Version 2.0\r\n", "")
```

Comment:
We should probably move this into a header clean function.  I can't imagine that this is the last malformed header we are going to see.

Reply:
We can't move it to a ``Form.clean_`` method, because ``_prepare_message`` is used to prepare the form data before initialization.

Reply:
I was thinking a custom list of frequent bad header lines that we filter out. Not necessarily using `Form.clean_` method.  The other option is to check for the EmailMessage defects, and if it is there punt on the message and add it as a failed message..

Reply:
Ah I understand, on it